File: mo-system/mo-system-api/src/main/java/ink/ckx/mo/system/api/model/entity/SysUser.java
package ink.ckx.mo.system.api.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import ink.ckx.mo.common.core.base.BaseEntity;
import ink.ckx.mo.common.web.enums.GenderEnum;
import ink.ckx.mo.common.web.enums.StatusEnum;
import lombok.Data;

@Data
public class SysUser extends BaseEntity {

    @TableId(type = IdType.ASSIGN_ID)
    private Long id;

    private String username;

    private String nickname;

    private GenderEnum gender;

    private String password;

    private Long deptId;

    private String avatar;

    private String mobile;

    private String email;

    private StatusEnum status;
}

File: mo-admin/mo-admin-api/src/main/kotlin/ink/ckx/mo/admin/api/model/entity/SysUser.kt
package ink.ckx.mo.admin.api.model.entity

import com.baomidou.mybatisplus.annotation.IdType
import com.baomidou.mybatisplus.annotation.TableId
import ink.ckx.mo.common.core.base.BaseEntity
import ink.ckx.mo.common.web.enums.GenderEnum
import ink.ckx.mo.common.web.enums.StatusEnum

data class SysUser(
    @TableId(type = IdType.ASSIGN_ID)
    var id: Long? = null,
    var username: String? = null,
    var nickname: String? = null,
    var gender: GenderEnum? = null,
    var password: String? = null,
    var deptId: Long? = null,
    var avatar: String? = null,
    var mobile: String? = null,
    var email: String? = null,
    var status: StatusEnum? = null,
) : BaseEntity()